Build.md

```diff
@@ -0,0 +1,123 @@
+
+The following commands have been tested with Powershell 5.1.
+
+Get the `neovim-qt` source code either via [git](https://git-scm.com/downloads)
+
+    > git clone https://github.com/equalsraf/neovim-qt.git
+    > cd neovim-qt
+
+or download the zip file.
+
+    > Invoke-WebRequest -Uri https://github.com/equalsraf/neovim-qt/archive/master.zip -OutFile master.zip
+    > Expand-Archive -Path master.zip .
+    > cd neovim-qt-master
+
+The easiest way to build `neovim-qt` on Windows 10 is to use the Qt installer (for example the [community edition](https://www.qt.io/download-open-source)).
+Download and run the Qt installer when asked which components to install select:
+
+- Qt _Version_
+    - MinGW _MinGW Version_
+- Developer and Designer Tools
+    - QT Creator
+    - MinGW _MinGW Version_
+    - CMake _Cmake Version_
+
+For example:
+
+![Select Qt Components](pics/MinGW-Qt-Components.png)
+
+After the successfull installation of Qt open a terminal (for example `powershell`) and navigate to the `neovim-qt` source code. 
+
+    > mkdir build
+    > cd build
+    > $env:Path += ";C:\Qt\Tools\CMake_64\bin\;C:\Qt\Tools\mingw730_64\bin\;C:\Qt\5.12.6\mingw73_64\bin\;C:\Program Files\Neovim\bin"
+    > cmake -DCMAKE_PREFIX_PATH="C:\Qt\5.12.6\mingw73_64\lib\cmake" -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=Release ..
+    > cmake --build .
+
+(These instructions assume the default installation path _C:\Qt_ for Qt, MinGW Version 7.3 and that the `neovim` binary is in _C:\Program Files\Neovim\bin_.)
+
+If the build was successfull, you can start `neovim-qt`.
+
+    > set NVIM_QT_RUNTIME_PATH=..\src\gui\runtime
+    > .\bin\nvim-qt
+
+
+## Windows 10 and Visual Studio 2019
+
+The following commands have been tested with Powershell 5.1.
+
+Download and install [`neovim`](https://github.com/neovim/neovim/wiki/Installing-Neovim) and [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/).
+
+Next download Qt (for example the [community edition](https://www.qt.io/download-open-source)). Run the Qt installer and when asked which components to install select:
+
+- Qt _Version_
+    - MSVC 2017 64-bit
+- Developer and Designer Tools
+    - QT Creator
+    - CMake _Cmake Version_
+
+For example:
+
+![Select Qt Components](pics/MSVC-Qt-Components.png)
+
+Finally, get the `neovim-qt` source code either via [git](https://git-scm.com/downloads)
+
+    > git clone https://github.com/equalsraf/neovim-qt.git
+    > cd neovim-qt
+
+or download the zip file.
+
+    > Invoke-WebRequest -Uri https://github.com/equalsraf/neovim-qt/archive/master.zip -OutFile master.zip
+    > Expand-Archive -Path master.zip .
+    > cd neovim-qt-master
+
+After the successfull installation of Qt open a terminal (for example `powershell`) and navigate to the `neovim-qt` source code. 
+
+    > mkdir build
+    > cd build
+    > $env:Path += ";C:\Qt\Tools\CMake_64\bin\;C:\Qt\5.12.6\msvc2017_64\bin\;C:\Program Files\Neovim\bin"
```

Comment:
nit: You could expand this to make the directions below easier to follow/explain.

For example in cmd (pardon my lack of powershell):
```
> set CMAKE_PATH=C:\Qt\Tools\CMake_64\bin
> set MSVC_PATH=C:\Qt\5.12.6\msvc2017_64\bin
> set NVIM_PATH=C:\Program Files\Neovim\bin

> set PATH=%PATH%;%CMAKE_PATH%;%MSVC_PATH%...
```

Reply:
I like that. I updated the instructions accordingly.